community/cypher/cypher-compiler-2.2_2.11/pom.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.neo4j</groupId>
+    <artifactId>cypher-parent</artifactId>
+    <version>2.2-SNAPSHOT</version>
```

Comment:
@boggle Seems I can't help but bug you: why no love for the parent element?

diff on compilers for 2.10 & 2.11:
```
neo4j/community/cypher$ diff -u cypher-compiler-2.2/pom.xml cypher-compiler-2.2_2.11/pom.xml |head -n 25
--- cypher-compiler-2.2/pom.xml	2015-04-09 10:07:47.795233100 +0200
+++ cypher-compiler-2.2_2.11/pom.xml	2015-04-09 10:07:47.825232937 +0200
@@ -3,17 +3,17 @@
   <parent>
     <groupId>org.neo4j</groupId>
     <artifactId>cypher-parent</artifactId>
-    <version>2.2.1-SNAPSHOT</version>
+    <version>2.2-SNAPSHOT</version>
     <relativePath>../</relativePath>
   </parent>
 
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.neo4j</groupId>
-  <artifactId>neo4j-cypher-compiler-2.2</artifactId>
+  <artifactId>neo4j-cypher-compiler-2.2_2.11</artifactId>
   <packaging>jar</packaging>
   <version>2.2.1-SNAPSHOT</version>
-  <name>Neo4j - Cypher Compiler 2.2</name>
-  <description>Compiler for Cypher 2.2</description>
+  <name>Neo4j - Cypher Compiler 2.2 for scala 2.11</name>
+  <description>Compiler for Cypher 2.2 for scala 2.11</description>
   <url>http://components.neo4j.org/${project.artifactId}/${project.version}</url>
```